lib/DDGC/DB/Result/InstantAnswer.pm

```diff
@@ -433,11 +440,39 @@ after insert => sub {
         meta2        => join('', map { sprintf ':%s:', $_ }
             $self->topics->columns([qw/ name /])->all),
         description  => sprintf('Instant Answer Page [%s](%s) created!',
-            $self->name, sprintf('https://duck.co/ia/view/%s',
-                $self->meta_id)),
+            $self->name, $self->uri( { activity_feed => 1 } ) ),
     } );
 };
 
+sub create_update_activity {
+    my ( $self, $meta3, $description ) = @_;
+    $self->result_source->schema->resultset('ActivityFeed')->updated_ia(
+        meta1        => $self->id,
+        meta2        => $self->topics->join_for_activity_meta( 'name' ),
+        meta3        => $meta3,
+        description  => $description,
+    );
+}
+
+sub _generate_updates {
+    my ( $self, $update ) = @_;
+
+    while ( my ($column, $value) = each $update ) {
+
+        if ( $column eq 'dev_milestone' ) {
```

Comment:
So we generate updates only when changing dev milestone? Good one, that's the most important field change so far. 
In the future I'd like to see all changes to important fields made by another user (not the one getting the notification mail).